src/Aquifer.API/Modules/Passages/PassagesModule.cs

```diff
@@ -60,108 +58,4 @@ private async Task<Results<Ok<List<PassagesByBookResponse>>, NotFound>> GetPassa
 
         return TypedResults.Ok(passagesByBook);
     }
-
-    private async Task<Results<Ok<PassageWithResourceItemsResponse>, NotFound>> GetPassageDetailsForLanguage(
```

Comment:
This endpoint was used a long time ago by Well but is no longer used. Had forgotten to remove it before.